doc/manuals/admin/perf_tuning.md

```diff
@@ -8,7 +8,8 @@
 * [Thread pool considerations](#thread-pool-considerations)
 * [Identifying bootlenecks looking at semWait statistics](#identifying-bootlenecks-looking-at-semwait-statistics)
 * [Log impact in performance](#log-impact-in-performance)
-* [Mutex policy impact in performance](#mutex-policy-impact-in-performance)
+* [Metrics impact in performance](#metrics-impact-in-performance)
```

Comment:
The conflict wasn't correctly resolved. Fixed in https://github.com/telefonicaid/fiware-orion/pull/2940/commits/f2c9efb8a381e531ffc9152a8c65410da21ff03e